### PR TITLE
[player-4067] Adding in chromecast button to the skin.

### PR DESCRIPTION
**Pull request by pilievOoyala**

### CODE REVIEW

**Comment by pilievOoyala** [comment]
This shouldn't affect anything until html5-skin's submodule is updated. Which will be soon

File: skin.json [hunk 1]
```diff
@@ -291,6 +291,7 @@
       {"name":"live", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":45},
       {"name":"timeDuration", "location":"controlBar", "whenDoesNotFit":"drop", "minWidth":145 },
       {"name":"flexibleSpace", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":1 },
+      {"name":"chromecast", "location": "controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":45 },
```
Comment (aeng7) [inline skin.json]: I think we should also add this to the buttonNames definition in the skin schema.

Comment (pilievOoyala) [reply skin.json]: done.

**Comment by aeng7** [comment]
LGTM

**Pull request merged**